ansible-st2-local/roles/arteria_core/defaults/main.yml

```diff
@@ -1,13 +0,0 @@
----
-
-arteria_disable_iptables: true
```

Comment:
Maybe this should be conditioned on `arteria_environment`. Either way we want to make sure this is not done in production.